tests/frame_test.go

```diff
@@ -72,3 +72,36 @@ func TestFrameDismissDialogBox(t *testing.T) {
 		})
 	}
 }
+
+func TestFrameNoPanicWithEmbeddedIFrame(t *testing.T) {
+	// This test only works when headless mode is false.
+	t.SkipNow()
```

Comment:
Maybe use `Skip()` and specify the reason there?

And only skip if `XK6_HEADLESS` (or `XK6_BROWSER_TEST_HEADLESS`?) is falsey?

Reply:
Good idea. I've added that in.

Resolved in: 2c88222b65722576ac3021eef97a6cb85703e860 and 9044d9a67609e56306994f351f7d22131f1e73b2